presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotQueryGeneratorContext.java

```diff
@@ -56,7 +56,11 @@
     public static final String TABLE_NAME_SUFFIX_TEMPLATE = "__TABLE_NAME_SUFFIX_TEMPLATE__";
     // Fields defining the query
     // order map that maps the column definition in terms of input relation column(s)
-    private final LinkedHashMap<VariableReferenceExpression, Selection> selections;
+    private final Map<VariableReferenceExpression, Selection> selections;
+    // Outputs are the fields defined in Presto for what's expected from Pinot Sql query.
+    // This is currently only used for Pinot Sql query code path.
```

Comment:
This line is stale: Outputs are now also used for the PQL code flow -- thank you !!!

Reply:
done.